scss/pages/_change-password.scss

```diff
@@ -0,0 +1,49 @@
+.change-password {
```

Comment:
Do we really need this much CSS rule? can we define and use some common elements for all pages instead?